test/secret/secret_m4_test.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("Secret Test for M4 SVA", func() {
 		}
 	})
 
-	Context("Multisite cluster deployment (M13 - Multisite indexer cluster, Search head cluster)", func() {
+	XContext("Multisite cluster deployment (M13 - Multisite indexer cluster, Search head cluster)", func() {
```

Comment:
Why are we disabling this?

Reply:
Accidental test code....... Reverting it.